src/services/apiService.ts

```diff
@@ -0,0 +1,51 @@
+import { checkError, handleError } from "../utils/errorHelper";
+import localStoreService from "./localStoreService";
+
+const apiRoute = "https://reqres.in/api";
+
+export function anonymPostRequest(route: string, body: unknown) {
```

Comment:
Can you give me more details why you create endpoints that do not need access token presence and the ones with ?

Reply:
I implemented all crud operations and their authorized versions to be able to write new functionality for authorized and non-authorized users